frontend/components/Storyline/Storyline.tsx

```diff
@@ -0,0 +1,21 @@
+// import { useState } from "react";
+// //http://localhost:8000/wt/api/nextjs/v1/pages/8/
+import RawHtml from "../RawHtml";
+import exampledata from "./exampledata.json";
+import StorylineScenario from "./StorylineScenario";
+
+export default function Storyline() {
+  return (
+    <>
+      <div className="flex flex-col lg:flex-row">
+        <h1>{exampledata.title}</h1>
+        <p>
+          <RawHtml html={exampledata.description} />
+        </p>
+      </div>
+      {exampledata.body.map((scenario, index) => (
+        <StorylineScenario key={index} storylineScenario={scenario} />
+      ))}
+    </>
+  );
+}
```

Comment:
This file is only referenced on the old homepage but gave me quite some confusion during the review. I think it's better to remove all references to the old storylines from the code.